roboteam_ai/src/skills/CoachDefend.cpp

```diff
@@ -11,13 +11,11 @@ CoachDefend::CoachDefend(std::string name, bt::Blackboard::Ptr blackboard)
         :Skill(std::move(name), std::move(blackboard)) { }
 
 void CoachDefend::onInitialize() {
-    coach::g_DefenceDealer.addDefender(robot->id);
-
     gtp.setCanMoveInDefenseArea(false);
 }
 
 bt::Node::Status CoachDefend::onUpdate() {
```

Comment:
why the name 'coach' in a skill?

Reply:
Because a normal 'defend' skill already existed and the difference should be clear. Regardless, this is not the PR to solve this problem in.

Reply:
As this has been on development for forever already